nginx.conf

```diff
@@ -0,0 +1,9 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    root /usr/share/nginx/html;
+    location / {
+        try_files $uri /index.html;
```

Comment:
```suggestion
        try_files $uri $uri/ /index.html;
```

Reply:
I think it's a best practice to serve static assets directly (with a specific route):

```
location /assets {
    # Serve static assets directly
    try_files $uri =404;
  }
```